dotCMS/src/main/java/com/dotcms/rest/api/v1/temp/TempFileAPI.java

```diff
@@ -217,17 +217,20 @@ public DotTempFile createTempFileFromUrl(final String incomingFileName,
    * @return
    * @throws IOException
    */
-  public boolean validUrl(final String url) throws IOException {
+  public boolean validUrl(final String url) {
 
     if(!(url.toLowerCase().startsWith("http://") ||
             url.toLowerCase().startsWith("https://"))){
       Logger.error(this, "URL does not starts with http or https");
       return false;
     }
-
-    final CircuitBreakerUrl urlGetter =
-            CircuitBreakerUrl.builder().setMethod(Method.GET).setUrl(url).build();
-    urlGetter.doString();
+    try {
+      final CircuitBreakerUrl urlGetter =
+              CircuitBreakerUrl.builder().setMethod(Method.GET).setUrl(url).build();
+      urlGetter.doString();
+    } catch (Exception e) {//If response is not 200, url is invalid
```

Comment:
We should handle the specific !=200 case instead of just catch any exception, we could be hiding real errors.

Reply:
The CircuitBreakerUrl throws BadRequestException or DotRuntimeException, so the only way could be catching both exceptions.